TKeazirian.HTTPServer.Tests/Router/RouterTests.cs

```diff
@@ -209,7 +191,6 @@ public void GetForSimpleOptionsReturns200Ok()
         Request testRequest = new Request("GET", "/method_options2", "", "");
         var testRoutesConfig = new RoutesConfig(new Dictionary<string, Handler>
         {
-            { "/test_path", new MockHandler() },
             { "/method_options2", new MockPostHandler() }
```

Comment:
why did you change from the generic '/test_path' here?  perhaps name it '/mock_post_path' ?